core/storage/suggestion/gae_models_test.py

```diff
@@ -2994,3 +2994,3029 @@ def test_get_model_association_to_user(self) -> None:
+                num_days_since_last_activity=None,
+                language_code='es'
+            ))
+        self.assertEqual(len(sorted_results), 4)
+        self.assertEqual(sorted_results[0].id, 'model_1')
+        self.assertEqual(sorted_results[1].id, 'model_2')
+        self.assertFalse(more)
+        self.assertEqual(next_offset, 4)
+
+        # Check for increasing Accuracy sort order.
+        sorted_results, next_offset, more = (
+            suggestion_models.TranslationSubmitterTotalContributionStatsModel
+            .fetch_page(
+                page_size=2,
+                offset=0,
+                sort_by=constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                            'IncreasingAccuracy'],
+                topic_ids=None,
+                num_days_since_last_activity=None,
+                language_code='es'
+            ))
+        self.assertEqual(len(sorted_results), 2)
+        self.assertEqual(sorted_results[0].id, 'model_4')
+        self.assertEqual(sorted_results[1].id, 'model_3')
+        self.assertTrue(more)
+        self.assertEqual(next_offset, 2)
+
+        # Check for decreasing Translated cards sort order.
+        sorted_results, next_offset, more = (
+            suggestion_models.TranslationSubmitterTotalContributionStatsModel
+            .fetch_page(
+                page_size=2,
+                offset=0,
+                sort_by=constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                            'DecreasingSubmissions'],
+                topic_ids=None,
+                num_days_since_last_activity=None,
+                language_code='es'
+            ))
+        self.assertEqual(len(sorted_results), 2)
+        self.assertEqual(sorted_results[0].id, 'model_3')
+        self.assertEqual(sorted_results[1].id, 'model_1')
+        self.assertTrue(more)
+        self.assertEqual(next_offset, 2)
+
+        # Check for increasing Translated cards sort order.
+        sorted_results, next_offset, more = (
+            suggestion_models.TranslationSubmitterTotalContributionStatsModel
+            .fetch_page(
+                page_size=2,
+                offset=0,
+                sort_by=constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                            'IncreasingSubmissions'],
+                topic_ids=None,
+                num_days_since_last_activity=None,
+                language_code='es'
+            ))
+        self.assertEqual(sorted_results[0].id, 'model_4')
+        self.assertEqual(sorted_results[1].id, 'model_2')
+        self.assertTrue(more)
+        self.assertEqual(next_offset, 2)
+
+        # Check for decreasing last activity sort order.
+        sorted_results, next_offset, more = (
+            suggestion_models.TranslationSubmitterTotalContributionStatsModel
+            .fetch_page(
+                page_size=2,
+                offset=0,
+                sort_by=constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                            'DecreasingLastActivity'],
```

Comment:
As mentioned in a previous review, this should be 4 spaces indented from the previous line. It is not.

Ditto elsewhere in several other places.

Reply:
I indented it after "=". Thanks for suggestio. Changed it.